components/tier-page/index.js

```diff
@@ -151,7 +151,7 @@ class TierPage extends Component {
     return (
       <div>
         <P fontSize="16px" color="black.700" fontWeight="bold" mt={4} mb={3}>
-          <FormattedMessage id="TierPage.ShareGoal" defaultMessage="Share this goal" />
+          <FormattedMessage id="TierPage.ShareGoal" defaultMessage="Share" />
```

Comment:
Let's make this one generic

```suggestion
          <FormattedMessage id="Share" defaultMessage="Share" />
```